docs/examples/compute/instance_chef/cookbooks/example_webserver/.kitchen.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I would be in favor of leaving this and the test case out, just to streamline the example.